recipes/telegram-bot/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "python-telegram-bot" %}
+{% set version = "9.0.0" %}
+{% set sha256 = "f5c3233bea7c7adf165e31225bbe9f28717e9f1f5070ebe99a4757c31c27ab28" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - certifi
+    - future >=0.16
+    # Optional:
+    - ujson
+    - pysocks
+
+test:
+  imports:
+    - telegram
+    - telegram.base
+    - telegram.ext
+
+about:
+  home: https://github.com/python-telegram-bot/python-telegram-bot
+  license: GPL-3.0
```

Comment:
From all of the meta-data and READMEs, it looks like the project switched to using LGPL:

https://pypi.org/project/python-telegram-bot/#license

If you agree, than please also change the `license_file` to `LICENSE.lesser`